packages/eas-cli/src/graphql/types/App.ts

```diff
@@ -2,7 +2,7 @@ import { Fragment } from '../fragment';
 
 export const AppFragment: Fragment = {
   name: 'app',
-  definition: `
+  definition: /* GraphQL*/ `
```

Comment:
```suggestion
  definition: gql`
```
(Maybe? I think we use `graphql-tag` in other places.)